Solutions/BitSight/Analytic Rules/BitSightPatchingCadenceDetected.yaml

```diff
@@ -0,0 +1,44 @@
+id: 53000762-286c-4fc6-b6fd-dca2338a38f1
+name: BitSight - patching cadence detected
+description: |
+  'Rule helps to detect whenever there is a patching cadence found in BitSight.'
+severity: Low
+status: Available
+requiredDataConnectors:
+  - connectorId: BitSight
+    dataTypes:
+      - BitSightFindingsData
+queryFrequency: 1d
+queryPeriod: 24h
+triggerOperator: GreaterThan
+triggerThreshold: 0
+tactics:
+  - Discovery
+query: |
+    let timeframe = 24h;
+    BitSightFindingsData
+    | where ingestion_time() > ago(timeframe)
+    | where RiskVector == "patching_cadence"
```

Comment:
This query is also a duplicate with RiskCategory == "Diligence".

Reply:
@devikamehra @v-atulyadav  We have removed the PatchingCadence analytics rule.
After internal discussion we have also removed the BitSightBadOpenPortsDetected analytics rule which would not be required as usecase from now.
Thanks.